msal_extensions/persistence.py

```diff
@@ -41,6 +45,15 @@ def _mkdir_p(path):
             raise
 
 
+# We do not aim to wrap every os-specific exception.
+# Here we define only the most common one,
+# otherwise caller would need to catch os-specific persistence exceptions.
+class PersistenceNotFound(OSError):
+    def __init__(
+            self, err_no=errno.ENOENT, message="Persistence not found", where=None):
```

Comment:
`where` seems a little confusing to me , can we use something like `location` or `persistence_location`?

Reply:
Admittedly, `where` is a weird name. The reason it was not named as `location` or `persistence_location` was that, in the macOS scenario, it is a combination of service_name and account_name, which I am not sure whether they fit into "location" concept. Maybe they do. Let me know. I am open for any suggestion here.

Reply:
After discussion, we agreed on `location`. Will change in next commit.